### PR TITLE
Create Action for format verification

### PR DESCRIPTION
**Pull request by vanyamil**

### CODE REVIEW

**Comment by vanyamil** [comment]
I recommend you enable Hide Whitespace during review:
image

File: Assets/Scripts/Characters/BarPatron.cs
Comment (NickoLefrench) [inline Assets/Scripts/Characters/BarPatron.cs:1]: Good idea to make the AI current action state

File: Assets/Scripts/Characters/PlayerInteraction.cs
Comment (NickoLefrench) [inline Assets/Scripts/Characters/PlayerInteraction.cs:1]: Interaction is more.sophisticated? It double checks the with what the player is interacting with

File: Assets/Scripts/Data/TunableEntry.cs
Comment (NickoLefrench) [inline Assets/Scripts/Data/TunableEntry.cs:1]: Unsure what was changed

File: README.md
Comment (NickoLefrench) [inline README.md:1]: So we all have to adjust visual studios to have proper formatting? Good idea for consistency

**Review by NickoLefrench — APPROVED** [review]

**Pull request merged**